views/partials/postHeader.handlebars

```diff
@@ -11,7 +11,7 @@
 		<time data-o-component="o-date" class="o-date" datetime="{{publishedDate}}">{{publishedDate}} </time>
 		<span class="meta-divider"> | </span>
 		{{#comments.enabled}}
-		<div class="comment-counter" data-o-component="o-comments" data-o-comments-count data-o-comments-config-article-id="{{id}}">Comments</div>
+		<div class="comment-counter" data-o-component="o-comments" data-o-comments-article-id="{{id}}" data-o-comments-count="true">Comments</div>
```

Comment:
I don't think this file is ever used and I was leaning towards deleting it...thoughts?